docs/POLYFILLS.md

```diff
@@ -45,6 +45,12 @@ import "core-js/es6";
 import "core-js/es7/reflect";
 import "core-js/es7/string";
 import "core-js/stage/4";
+/**
+ * IE11 does not support iteration on certain DOM collections (NodeList).
+ * This polyfill is specifically needed for the animation on mat-menu used in stark-table.
+ * [More info]{@link https://github.com/angular/angular/issues/27887}
```

Comment:
Can you remove the MD syntax of this comment... it is not displayed correctly. Just leave it like this:

```
More info: https://github.com/angular/angular/issues/27887
```